Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityRaycastProvider.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Physics;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Interface to handle raycasts into the scene.
```

Comment:
Add a bit more information to make it clear to implementers where this interface is used by MRTK. I'd at least mention that FocusProvider will use it to perform raycast and sphere cast queries for pointers.

Reply:
Added a blurb about FocusProvider.